api/firmware/device.go

```diff
@@ -332,11 +332,17 @@ func (device *Device) Product() common.Product {
 }
 
 // SupportsETH returns true if ETH is supported by the device api.
-// coinCode is eth/teth/reth or eth-erc20-xyz, ...
+// coinCode is eth/teth/reth/goeth or eth-erc20-xyz, ...
```

Comment:
uuh delete this line? `coinCode` is not a parameter here (probably was in the past)

Reply:
ACK